packages/botonic-react/src/webchat/header.jsx

```diff
@@ -42,15 +42,18 @@ const StyledHeaderTitle = styled(Flex)`
 `
 
 export const DefaultHeader = props => {
+  const { webchatState, useTheme } = props
+  const { theme } = webchatState
   let HeaderImage = Logo
-  if ('brandImage' in props.webchatState.theme)
-    HeaderImage = props.webchatState.theme.brandImage
-  if ('headerImage' in props.webchatState.theme)
-    HeaderImage = props.webchatState.theme.headerImage
+  // These conditions are done in this way because HeaderImage can be null
+  if (getProperty(theme, 'brand.image') || getProperty(theme, 'brandImage'))
+    HeaderImage = useTheme('brand.image')
```

Comment:
should this be BrandImage = useTheme('brand.image') ?

Reply:
HeaderImage := DefaultLogo <-- This is the one we set by default
if user has brand.image or brandImage --> HeaderImage := brand.image or brandImage
if user has header.image or headerImage --> HeaderImage := header.image or headerImage 

In both cases, both brand.image and header.image are replacing the image in the header, so this is why it's done in this way. At the same time, header.image has preference over brand.image.